cypress/e2e/app/recommendations/filter-summary-records.cy.ts

```diff
@@ -155,12 +155,80 @@ describe("Recommendations", () => {
         );
         cy.get(".filters-drawer-container .mat-drawer-opened").should("exist");
         cy.wait(2000);
-        cy.get("td.cdk-column-designatedBody").each(($el, index, $list) => {
+        cy.get("td.cdk-column-designatedBody").each(($el) => {
           expect($el.text() === "1-AIIDWT").to.be.false;
         });
       });
     });
 
+    describe("Filter by GMC status", () => {
+      it("should display 'gmc status' selection list filters", () => {
+        initFilterPanel();
+        cy.get("[data-cy='selectionListgmcStatus']").should("exist");
+      });
+
+      it("should display 3 selection options in the 'gmc status' selection list labelled accordingly", () => {
+        const gmcStatusLabels = ["Approved", "Rejected", "Under Review"];
+
+        initFilterPanel();
+
+        cy.get(
+          "[data-cy='selectionListgmcStatus'] .mat-list-option .mat-list-text"
+        ).should("have.length", 3);
+        cy.get(
+          "[data-cy='selectionListgmcStatus'] .mat-list-option .mat-list-text"
+        ).each(($el) => {
+          expect(gmcStatusLabels.indexOf($el.text().trim())).to.be.greaterThan(
+            -1
+          );
+        });
+      });
+
+      it("should filter summary records list by single gmc status when single option selected and submitted", () => {
+        initFilterPanel();
+
+        cy.get("[data-cy='selectionOptionApproved']").should("exist");
+        cy.get("[data-cy='selectionOptionApproved']").click();
+        checkButtonsDisabled("not.be.disabled");
+
+        submitForm();
+
+        cy.get("td.cdk-column-gmcOutcome").each(($el) => {
+          expect($el.text()).to.equal("Approved");
+        });
+      });
+
+      it("should filter summary records list by multiple gmc statuses when multiple options selected and submitted", () => {
+        initFilterPanel();
+        const selectedGmcStatus = ["Approved", "Rejected"];
+        selectedGmcStatus.forEach((status) => {
+          const $el = cy.get("[data-cy='selectionOption" + status + "']");
+          $el.should("exist");
+          $el.click();
+        });
+
+        checkButtonsDisabled("not.be.disabled");
+
+        submitForm();
+
+        const gmcStatus: string[] = [];
+        cy.get("td.cdk-column-gmcOutcome")
+          .each(($el) => {
+            gmcStatus.push($el.text());
+          })
+          .then(() => {
+            expect(
+              gmcStatus.every((status) => {
+                return (
+                  status === selectedGmcStatus[0] ||
+                  status === selectedGmcStatus[1]
+                );
+              })
+            ).to.be.true;
+          });
```

Comment:
This seems like a different way of doing `expect($el.text().trim()).to.be.oneOf(selectedGmcStatus);` in another test.  If so, I expect it should be implemented in the same way.